docs/source/Tutorial.md

```diff
@@ -1134,6 +1399,14 @@ before:
   }
 ~~~
 </div>
+<div class="language-c">
+~~~{.c}
+  #include "monster_reader.h"
+
+  #define ns(x) FLATBUFFERS_WRAP_NAMESPACE(MyGame_Sample, x) // Specified in the schema.
+  #define nsc(x) FLATBUFFERS_WRAP_NAMESPACE(flatbuffers, x)
```

Comment:
I'd say it is more readable without the macro, since the macro results in weird looking code allover.